### PR TITLE
Update submit training operator job and charts.

### PR DESCRIPTION
**Pull request by Syulin7**
1.update submit training operator job. 
2.update tfjob,pytorchjob charts.
3.update mpijob charts and upgrade mpijob apiVersion to v1.
4.code clean.

### CODE REVIEW

File: charts/pytorchjob/templates/pytorchjob.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ metadata:
   {{- end }}
 spec:
 {{- if .Values.cleanPodPolicy }}
-  cleanPodPolicy: {{ .Values.cleanPodPolicy }}
```
Comment (cheyang) [inline charts/pytorchjob/templates/pytorchjob.yaml:24]: please also update chart version and changelog.

Comment (Syulin7) [reply charts/pytorchjob/templates/pytorchjob.yaml:24]: Done

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ version: 2
 jobs:
   build:
     docker:
-    - image: circleci/golang:1.14.10
+    - image: cimg/go:1.19.4
```
Comment (cheyang) [inline .circleci/config.yml:8]: why not using circleci/golang?

Comment (Syulin7) [reply .circleci/config.yml:8]: circleci/golang is no longer updated.
cimg/go is designed to supercede the legacy CircleCI Go image, circleci/golang.
see:  https://github.com/CircleCI-Public/cimg-go

Comment (Syulin7) [reply .circleci/config.yml:8]: circleci working_directory has changed from /go to /home/circleci/go.

**Pull request merged**